File: INICIANTE/JAVASCRIPT/1036.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var prompt = function(texto) { return lines.shift(); };

var [A, B, C] = prompt("A").split(" ").map(Number)

var D = Math.pow(B , 2) - (4*A*C)
var x1 = (-B - Math.sqrt(D)) / (2*A)
var x2 = (-B + Math.sqrt(D)) / (2*A)
if(D < 0 || A === 0){
    console.log("Impossivel calcular")
}
else{
    console.log("R1 = " + x2.toFixed(5))
    console.log("R2 = " + x1.toFixed(5))
}

File: INICIANTE/JAVA/1036.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        double A, B, C, D, x1, x2;
        Scanner sc = new Scanner(System.in);
        A = sc.nextDouble();
        B = sc.nextDouble();
        C = sc.nextDouble();
        D = Math.pow(B, 2) - (4 * A * C);
        x1 = (-B - Math.sqrt(D)) / (2 * A);
        x2 = (-B + Math.sqrt(D)) / (2 * A);

        if (D < 0 || A == 0) {
            System.out.print("Impossivel calcular" + "\n");
        } else {
            System.out.println(String.format("R1 = %.5f", x2));
            System.out.println(String.format("R2 = %.5f", x1));
        }
        sc.close();
    }
}
